packages/react-composites/tests/browser/call/hermetic/VideoGallery.test.ts

```diff
@@ -255,6 +258,7 @@ test.describe('VideoGallery tests', async () => {
 
     await waitForSelector(page, dataUiId(IDS.moreButton));
     await pageClick(page, dataUiId(IDS.moreButton));
+    await page.locator('span:has-text("View")');
```

Comment:
Is this needed?

Reply:
Or should the below line be `await (await page.locator('...')).click()`

Reply:
just to help with flakeyness seems to get too eager if it only has the one. I think we need to take another look at the mobile snapshots and interacting with the bottom sheet drawer since we don't use the data-ui-id's there.

Reply:
Hmm, yeah dont need to fix in this PR, but if thats the case this will just hide the flakiness more and not garaunteed to fix it

Reply:
Going to disable auto merge in case you need to change this @dmceachernmsft